upstairs/src/upstairs.rs

```diff
@@ -1858,7 +1885,7 @@ impl Upstairs {
         self.on_reconciliation_skipped()
     }
 
-    /// Checks whether we can connect all three regions
+    /// Response to a client entering `NegotiationState::WaitQuorum`
     ///
     /// Returns `false` if we aren't ready, or if things failed.  If there's a
     /// failure, then we also update the client state.
```

Comment:
I'd vote for returning `Result<bool, some error>` instead - having `false` cover both cases seems wrong

Reply:
Counterproposal: let's remove the return value entirely!  It's only ever used in one unit test, so I've updated that test to look at states instead.

﻿fb2b63285050